src/MuxManager.cpp

```diff
@@ -570,6 +570,12 @@ void MuxManager::handleWarmRestartReconciliationTimeout(const boost::system::err
         MUXLOGWARNING("Reconciliation timed out after warm restart, set service to reconciled now.");
     }
 
+    uint32_t rc = system("sudo config muxcable mode auto all"); // rc > 0 means success
+
+    if (rc <= 0) {
```

Comment:
Are we sure that rc>0 is a success case? Since here system command is executed, and in Linux system return code > 0 is error.

Reply:
I was referring to the case when child process could not be created (rc==0 or rc==-1). And log was intended for debugging purpose only. 

Guess the wording is misleading. Will update. 

reference: https://man7.org/linux/man-pages/man3/system.3.html